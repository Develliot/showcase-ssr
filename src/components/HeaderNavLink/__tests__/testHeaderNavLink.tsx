import React from 'react';
import { shallow, ShallowWrapper } from 'enzyme';
import { HeaderNavLink } from '../';

it('renders the component properly with all of its props', () => {
    const wrapper: ShallowWrapper = shallow(
        <HeaderNavLink selected={false} title={'title'} to={'/page'} />
    );
    expect(wrapper).toMatchSnapshot();
});

it('renders the component properly when selected', () => {
    const wrapper: ShallowWrapper = shallow(
        <HeaderNavLink selected={false} title={'title'} to={'/page'} />
    );
    expect(wrapper).toMatchSnapshot();
});
