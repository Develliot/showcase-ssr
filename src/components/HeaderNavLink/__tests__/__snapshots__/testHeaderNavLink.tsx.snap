// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the component properly when selected 1`] = `
<styles__HeaderLinkItem>
  <Link
    href="/page"
  >
    <styles__NavLinkStyled
      title="title"
    >
      <Typography__Paragraph
        color="white"
      >
        title
      </Typography__Paragraph>
      <Layout__RowCenter>
        <styles__HoverInidcator
          selected={false}
        />
      </Layout__RowCenter>
    </styles__NavLinkStyled>
  </Link>
</styles__HeaderLinkItem>
`;

exports[`renders the component properly with all of its props 1`] = `
<styles__HeaderLinkItem>
  <Link
    href="/page"
  >
    <styles__NavLinkStyled
      title="title"
    >
      <Typography__Paragraph
        color="white"
      >
        title
      </Typography__Paragraph>
      <Layout__RowCenter>
        <styles__HoverInidcator
          selected={false}
        />
      </Layout__RowCenter>
    </styles__NavLinkStyled>
  </Link>
</styles__HeaderLinkItem>
`;
